runtime/src/main/java/org/corfudb/util/CorfuComponent.java

```diff
@@ -7,10 +7,14 @@
  * Created by Sam Behnam on 5/8/18.
  */
 public enum CorfuComponent {
+    // Runtime components
     ASV("corfu.runtime.as-view."),
```

Comment:
I'm not sure if this is a convention, but its not readable. Can you expand to at least 1-2 words?

Reply:
It was like this before I refactored them to an enum but non readability is good point. Made the change as suggested.